src/components/ChartRow.js

```diff
@@ -151,18 +144,31 @@ export default class ChartRow extends React.Component {
 
                 const scale = createScale(child, type, min, max, rangeBottom, rangeTop);
                 if (!_.has(this.scaleMap, id)) {
-                    // No scale map yet, create one on this.state.yAxisScalarMap
+                    // If necessary, initialize a ScaleInterpolator for this y-axis.
+                    // When the yScale changes, we will update this interpolator.
                     this.scaleMap[id] = new ScaleInterpolator(transition, easeSinOut, s => {
                         const yAxisScalerMap = this.state.yAxisScalerMap;
                         yAxisScalerMap[id] = s;
                         this.setState(yAxisScalerMap);
                     });
                 }
+                // Get the vertical scale for this y-axis.
+                let scale;
```

Comment:
scale is defined above as well. Line 145

Reply:
It looks like something went wrong with the recent merge from master. If you look at the [previous state of the branch ](https://github.com/ethanroday/react-timeseries-charts/blob/61ec0ad268f1d020f121330699fd8d38538071b1/src/components/ChartRow.js), this issue doesn't appear.

Reply:
Ah, yeah i believe that's the case :(